_episodes/05-loop.md

```diff
@@ -362,12 +362,31 @@ from whatever file is being processed
 > ```
 > {: .output}
 > Try removing the quotes around `$filename` in the loop above to see the effect of the quote
-> marks on whitespace:
+> marks on whitespace. Note that we get a result from the loop command for unicorn.dat if you ran this code in the creatures directory:
```

Comment:
Grammar nitpick: how about "when we run this code" instead of "if you ran this code" (to be consistent with the beginning of the sentence).